pkg/downloader/downloader_test.go

```diff
@@ -130,3 +131,29 @@ func TestDownloadLocal(t *testing.T) {
 	})
 
 }
+
+func TestDownloadCompressed(t *testing.T) {
+
+	if runtime.GOOS == "windows" {
+		// FIXME: `assertion failed: error is not nil: exec: "gzip": executable file not found in %PATH%`
```

Comment:
We can use stdlib for gzip.
Can be another PR in the future

Reply:
I'm not sure if anyone is using gzip, because I think the image format supports this natively ? ("stream-optimized")